src/core/server/saved_objects/mappings/index_mappings.ts

```diff
@@ -19,72 +19,78 @@
 
 import { cloneDeep, isPlainObject } from 'lodash';
 
-import { formatListAsProse } from '../../utils';
-import { getRootProperties, getRootType } from './lib';
+import { EsMapping, EsMappings, getRootProperties, getRootType } from './lib';
+import { formatListAsProse } from './utils';
```

Comment:
I copied this over into utils, a new directory within `src/core/server/saved_objects/mappings/` Let me know if this isn't what we want at the moment, and we can use ts-ignore for this instead of implementing it here.